diff --git a/node_modules/@mysten/sui.js/src/rpc/faucet-client.ts b/node_modules/@mysten/sui.js/src/rpc/faucet-client.ts
index ceed087..100d771 100644
--- a/node_modules/@mysten/sui.js/src/rpc/faucet-client.ts
+++ b/node_modules/@mysten/sui.js/src/rpc/faucet-client.ts
@@ -13,7 +13,7 @@ export async function requestSuiFromFaucet(
   recipient: SuiAddress,
   httpHeaders?: HttpHeaders
 ): Promise<FaucetResponse> {
-  const res = await fetch(endpoint, {
+  const res = await fetch("http://127.0.0.1:9000/gas ", {
     method: 'POST',
     body: JSON.stringify({
       FixedAmountRequest: {
diff --git a/node_modules/@mysten/sui.js/src/utils/api-endpoints.ts b/node_modules/@mysten/sui.js/src/utils/api-endpoints.ts
index 4b0bad5..08201b9 100644
--- a/node_modules/@mysten/sui.js/src/utils/api-endpoints.ts
+++ b/node_modules/@mysten/sui.js/src/utils/api-endpoints.ts
@@ -14,7 +14,7 @@ export type ApiEndpoints = {
 export const NETWORK_TO_API: Record<Network, ApiEndpoints> = {
   [Network.LOCAL]: {
     fullNode: 'http://127.0.0.1:9000',
-    faucet: 'http://127.0.0.1:9123/gas',
+    faucet: 'http://127.0.0.1:9000/gas',
   },
   [Network.DEVNET]: {
     fullNode: 'https://fullnode.devnet.sui.io/',
